addTag.php

```diff
@@ -0,0 +1,283 @@
+                <option value="MI">Michigan</option>
+                <option value="MN">Minnesota</option>
+                <option value="MS">Mississippi</option>
+                <option value="MO">Missouri</option>
+                <option value="MT">Montana</option>
+                <option value="NE">Nebraska</option>
+                <option value="NV">Nevada</option>
+                <option value="NH">New Hampshire</option>
+                <option value="NJ">New Jersey</option>
+                <option value="NM">New Mexico</option>
+                <option value="NY">New York</option>
+                <option value="NC">North Carolina</option>
+                <option value="ND">North Dakota</option>
+                <option value="OH">Ohio</option>
+                <option value="OK">Oklahoma</option>
+                <option value="OR">Oregon</option>
+                <option value="PA">Pennsylvania</option>
+                <option value="RI">Rhode Island</option>
+                <option value="SC">South Carolina</option>
+                <option value="SD">South Dakota</option>
+                <option value="TN">Tennessee</option>
+                <option value="TX">Texas</option>
+                <option value="UT">Utah</option>
+                <option value="VT">Vermont</option>
+                <option value="VA" selected>Virginia</option>
+                <option value="WA">Washington</option>
+                <option value="WV">West Virginia</option>
+                <option value="WI">Wisconsin</option>
+                <option value="WY">Wyoming</option>
+            </select>
+
+            <label for="zip"><em>* </em>Zip Code</label>
+            <input type="text" id="zip" name="zip" pattern="[0-9]{5}" title="5-digit zip code" required placeholder="Enter your 5-digit zip code">
+
+
+            <label for="opnotes">Operation Notes</label>
+            <textarea wrap="soft" id="opnotes" name="opnotes" placeholder="Input operation notes"></textarea>
+
+
+            <label for="adtl-services">Additional Services Offered</label>
+            <input type="text" id="adtl-services" name="adtl-services">
+
+            <label for="tag"><em>* </em>Tag</label>
+            <select id="tag" name="tag" required>
+                <option value="">Choose an option</option>
+                <option value="Male">Male</option>
+                <option value="Female">Female</option>
+                <option value="Other">Other</option>
+            </select>
+
+
+        </fieldset>
+
+    
+        <fieldset>
+            <legend>Food Bank Schedule</legend>
+
+            <br>
+
+            <p>Please enter the days, times, and frequency of the food bank.</p>
+
+            
+            <label><em>* </em>Availability</label>
+            <p>Enter the days and times the food bank is available each week.</p>
+
+            <div class="availability-container">
+
+                <div class="availability-day">
```

Comment:
The radio buttons turn on and off, but not the drop-downs do not activate regardless.

This could be fixed with a PHP if statement to the effect of:
$checked=$_POST['radio_btn_var']
if $checked!=NULL:
     $drop_down_var remove hidden (you may be able to use prop or attr, but I'm not entirely sure)